packages/http-client/src/auth.ts

```diff
@@ -23,9 +23,9 @@ export class BasicCredentialHandler implements ifm.IRequestHandler {
   handleAuthentication(
     httpClient: ifm.IHttpClient,
     requestInfo: ifm.IRequestInfo,
-    objs
+    objs: any
   ): Promise<ifm.IHttpClientResponse> {
-    return null
+    throw new Error("not implemented")
```

Comment:
What do these changes do?

Reply:
Returning `null` fails type checking with `strict: true`.  I could have just added `| null` to the return type, but that doesn't make sense for when the method is actually implemented.  So better to throw an exception for when it's not implemented.  Callers can use `canHandleAuthentication()` to make sure the method is implemented.